tmux/paths.go

```diff
@@ -0,0 +1,35 @@
+package tmux
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// alternatePath returns an altnerate string that should be check when doing
+// path based comparisons.
+func alternatePath(s string) (altPath string) {
```

Comment:
This whole function seems useful for the connect command as well, being able to use relative paths to create a session.

```
sesh connect ./sub/dir`
``

Can we reuse this logic in the connect function?

Reply:
I'm not sure what the ask is here.

The `connect` function already uses this function indirectly, when it checks for an existing session. With these changes the `connect` command also works with arbitrary paths (fully qualified, sub-dir, and relative paths). 

What else are you looking to have done here?

Reply:
Instead of just asking for an existing session I'd like to be able to use the same pattern to create new sessions.

Reply:
Unless I am missing something, with this PR, it already works the way you are describing. Some examples below.


<details>
<summary>baseline</summary>
<p>

```
❯ zoxide query -a -l

❯ go run *.go list
sesh
```

</p>
</details>

<details>
<summary>create sessions</summary>
<p>

```
❯ go run *.go connect ~/dotfiles
❯ go run *.go connect ../../
❯ mkdir pr57; go run *.go connect pr57
```

</p>
</details>

<details>
<summary>results</summary>
<p>

```
❯ go run *.go list
sesh
pr57
dev
dotfiles
❯ zoxide query -a -l
/Users/mark/dev/sesh
/Users/mark/dev/sesh/fix_rename/pr57
/Users/mark/dev
/Users/mark/dotfiles
```

</p>
</details>